components/printMessages/printMessages.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { StyledSubHeader } from '../../pageStyles/emailSender.styles'
+import { SectionContainer, StyledButton } from '../../styles/common'
+
+type PrintMessageProps = {
+    length: number,
+    createMessages: any,
+
+}
+
+export default function PrintMessage ({ length, createMessages } :PrintMessageProps) {
```

Comment:
Lets name this to PrintMessages since the file name is printMessages.tsx (same with the props)